src/component/Collapse.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import styled from 'styled-components';
+
+const CWrapper = styled.div`
+    display: inline;
+    width: 100%;
+`
+
+const CTitle = styled.div`
+    float: right;
+    font-size: 12px;
+    color: #c6c6c6;
+`
+const CContent = styled.div`
+    font-size: 13px;
+    color: #c6c6c6;
+`
+
+class Collapse extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            open: false
+        }
+        this.togglePanel = this.togglePanel.bind(this);
+    }
+
+    togglePanel(e){
+        this.setState({open: !this.state.open})
+    }
+
+    render() {
+        return (
+            <CWrapper>
```

Comment:
이름 너무 안줄여도 괜찮아ㅋㅋㅋㅋㅋㅋㅋ
```suggestion
            <CollapseWrapper>
```
정도로 바꿔주면 좋을 것 같아